### ISSUE TITLE
SPVCNN Hyperparameters

### ISSUE THREAD

**Issue by CCInc** [mit-han-lab/e3d#9]
Hi,

The parameters in `configs/semantic_kitti/default.py` are designed for the SPVNAS network, correct? I'm curious about the parameters used for the SPVCNN and MinkUNet models, in particular the learning rate. I couldn't find it in the paper.

Thanks! Congratulations on your great work.

**Comment by kentangSJTU** [comment]
Hi,

Thanks for your interest in our work. Actually these hyper parameters are for MinkUNet and SPVCNN. We haven't released SPVNAS super network training / architecture finetuning config files, but these parameters are indeed given in the paper appendix.

Best,
Haotian

**Comment by CCInc** [comment]
I see, perhaps there is something wrong with my implementation then. Thank you for your help.
